app-web/src/pages/topics_table_of_Content.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import withResourceQuery from '../hoc/withResourceQuery';
+import { flattenGatsbyGraphQL } from '../utils/dataHelpers';
+import Layout from '../hoc/Layout';
+import { Title } from '../components/Page';
+import Main from '../components/Page/Main';
+import TableofContents from '../components/TopicPreview/TableofContents';
+import styled from '@emotion/styled';
+
+const OutsideBorder = styled.li`
+  top-margin: 15px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.5);
+`;
+
+const AccordionList = styled.ul`
+  margin: 10px;
+  padding: 10px;
+  background-color: #fff;
+  max-width: 650px;
+  overflow: scroll;
+  border-top: 1px solid rgba(#000, 0.1);
+`;
+
+export const TopicsTableofContents = ({ data }) => {
+  let topics = flattenGatsbyGraphQL(data.allDevhubTopic.edges);
+  return (
+    <Layout>
+      <Main>
+        <Title title={'Table of Content for Topics'} />
+        <AccordionList>
+          {topics.map(topic => (
+            <OutsideBorder key={topic.id}>
+              <TableofContents key={topic.id} title={topic.name} resources={topic.connectsWith} />
+            </OutsideBorder>
+          ))}
+        </AccordionList>
+      </Main>
+    </Layout>
+  );
+};
+export default withResourceQuery(TopicsTableofContents)();
```

Comment:
wrong query to use, its best to make a custom query for this page. withResourceQuery returns a ton of data that is not used in this page! 

https://www.gatsbyjs.org/tutorial/part-four/#your-first-graphql-query